lib/src/assemble/manifest.rs

```diff
@@ -447,27 +444,38 @@ impl<'a> FakeDataAttributes<'a> {
                 );
             }
 
-            // hubris artifacts: build a fake archive (SimGimletSp and
+            // In production, all the bootloaders and RoTs claim to have the
+            // same board (currently: `oxide-rot-1`). Let's do that here too.
+            KnownArtifactKind::GimletRotBootloader
+            | KnownArtifactKind::PscRotBootloader
+            | KnownArtifactKind::SwitchRotBootloader
+            | KnownArtifactKind::GimletRot
+            | KnownArtifactKind::PscRot
+            | KnownArtifactKind::SwitchRot => "SimRot",
```

Comment:
No need to change anything here, but just curious. What are the consequences of using the real board names here instead of "SimRot",  "SimPscSp", etc?

Reply:
We'd have to change the `sp-sim` to match for tests to work, I think. I could see an argument for that, but to date we've kept them separate. I _think_ it shouldn't matter much either way, except that it's nice to be able to immediately tell whether something is fake?